app/generators/questionnaire_generator.rb

```diff
@@ -306,15 +326,19 @@ def checkbox_group_klasses(question)
 
     def checkbox_options(question)
       body = []
-      question[:options].each do |option|
+      question[:options].each_with_index do |option, idx|
```

Comment:
dit stuk is hetzelfde als wat er op regel 165 gebeurt. En aangezien het nou niet per se de mooiste code is, misschien kunnen we het samenvoegen?

Reply:
Fixed.